examples/postgres-RDS/provision-rds-blueprint.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: rds-postgres-bp
+actions:
+  backup:
+    kind: StatefulSet
+    outputArtifacts:
+      cloudObject:
+        keyValue:
+          backupLocation: "{{ .Phases.pgDump.Output.backupLocation }}"
+    phases:
+    - func: KubeTask
+      name: pgDump
+      objects:
+        pgSecret:
+          kind: Secret
+          name: '{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-postgresql'
+          namespace: '{{ .StatefulSet.Namespace }}'
+      args:
+        image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
+        namespace: '{{ .StatefulSet.Namespace }}'
+        command:
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          export PGHOST='{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-postgresql.{{ .StatefulSet.Namespace }}.svc.cluster.local'
+          export PGUSER='postgres'
+          export PGPASSWORD='{{ index .Phases.pgDump.Secrets.pgSecret.Data "postgres-password" | toString }}'
+          BACKUP_LOCATION=pg_backups/{{ .StatefulSet.Namespace }}/{{ .StatefulSet.Name }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time | date "2006-01-02T15:04:05Z07:00" }}/backup.sql.gz
+          pg_dumpall --clean -U $PGUSER | gzip -c | kando location push --profile '{{ toJson .Profile }}' --path "${BACKUP_LOCATION}" -
+          kando output backupLocation "${BACKUP_LOCATION}"
+  restore:
+    kind: StatefulSet
+    inputArtifactNames:
+    - cloudObject
+    phases:
+    - func: KubeTask
+      name: pgRestore
+      args:
+        image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
+        namespace: '{{ .StatefulSet.Namespace }}'
+        command:
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          # Set the region
+          export REGION="{{ .ConfigMaps.dbconfig.Data.region | toString }}"
+
+          export INSTANCE_NAME="{{ .ConfigMaps.dbconfig.Data.instance_name | toString }}"
+
+          # get the pod IP
+          POD_IP=$(hostname -i)
+
+          export accessKeyId="{{ .Secrets.dbsecret.Data.accessKeyId | toString }}"
+          export secretAccessKey='{{ .Secrets.dbsecret.Data.secretAccessKey | toString }}'
+
+          ####Configure AWS
+          aws configure set aws_access_key_id $accessKeyId
+          aws configure set aws_secret_access_key $secretAccessKey
+
+          # Set the default region
+          aws configure set default.region $REGION
+
+          # Set the default output format
+          aws configure set default.output json
+          
+          # SG creation
+          SG_NAME=rds-security-group
+          SG_DESC="rds-Security-group-from-Blueprint"
```

Comment:
I think we don't necessarily need these comments. Because, by adding these type of comments, we are not giving any extra information that readers can't get by just looking at the script itself. for example in below case

```
# SG creation
SG_NAME=rds-security-group
SG_DESC="rds-Security-group-from-Blueprint"
```

In this above case `SG_NAME=rds-security-group` line provides more details than the comment. Let's remove the comments that are not useful.